### DISCUSSION TITLE
sh: 1: nuxt: Permission denied

### DISCUSSION THREAD

**Discussion by tanvirismail** [nuxt/nuxt#9592] [Questions (Nuxt 2)]
My AWS Ubuntu server does not have permission to run nuxt command. I use this command `npm run build`, please help me. 

sh: 1: nuxt: Permission denied
npm ERR! code ELIFECYCLE
npm ERR! errno 126
npm ERR! netfior@2.0.0 build: `nuxt build`
npm ERR! Exit status 126
npm ERR!
npm ERR! Failed at the netfior@2.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-07-24T03_13_03_674Z-debug.log

**Comment by tanvirismail** [comment]
ok, i got the solution `chmod -R 0777 node_modules`

**Reply by kissu** [reply]
This is probably not a real solution because of security reasons.

**Comment by HenrijsS** [comment]
I suggest you set proper permission for the whole `serve` directory and avoid giving 777 permission for any file or potential executable
